src/Entity/AdherentMandate/AbstractAdherentMandate.php

```diff
@@ -103,6 +103,11 @@ abstract class AbstractAdherentMandate
      */
     private $reason;
 
+    /**
+     * @ORM\Column(type="boolean", options={"default": false})
+     */
+    public $provisional = false;
```

Comment:
Because I need access this field event from `AbstractAdherentMandate`.
For exemple, if this field is only in `CommitteeAdherentMandate`, an error is thrown in this method https://github.com/EnMarche/en-marche.fr/pull/5275/files#diff-b7ec04edeabafcfd4a82006f3248fc460d8dd57079979f04c7b07f7bd7a35c36R1868